packages/components/src/components/Calendar/style.tsx

```diff
@@ -0,0 +1,193 @@
+import styled, { css } from '../../utility/styled';
+import { colors } from '../../variables';
+import { bodyFont } from '../../variables';
+import Box from '../Box';
+
+const calendarCssStyles = css`
+    button {
+        color: inherit;
+        font-family: inherit;
+        font-size: inherit;
+    }
+
+    .react-calendar {
+        width: 280px;
+        max-width: 100%;
+        background: white;
+        font-family: ${bodyFont} !important;
+        font-style: normal;
+        font-weight: 400;
+        font-size: 15px;
+        line-height: 24px;
+        color: ${colors.grey600};
```

Comment:
Other text components use font-smoothing which comes closer to Figma font rendering. It isn't standardized though, so in this rare case you can use prefixes:
```suggestion
        color: ${colors.grey600};
        -webkit-font-smoothing: antialiased;
        -moz-osx-font-smoothing: grayscale;
```